app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,316 @@
+            app:layout_constraintEnd_toEndOf="@id/right_guidline"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/tv_movie_announce" />
+
+        <ImageView
+            android:id="@+id/iv_cast_image_1"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="7dp"
+            android:adjustViewBounds="true"
+            android:src="@drawable/image_1"
+            app:layout_constraintEnd_toStartOf="@+id/iv_cast_image_2"
+            app:layout_constraintHorizontal_chainStyle="spread"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/tv_movie_cast" />
+
+        <ImageView
+            android:id="@+id/iv_cast_image_2"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_2"
+            app:layout_constraintEnd_toStartOf="@+id/iv_cast_image_3"
+            app:layout_constraintStart_toEndOf="@+id/iv_cast_image_1"
+            app:layout_constraintTop_toTopOf="@id/iv_cast_image_1" />
+
+        <ImageView
+            android:id="@+id/iv_cast_image_3"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_3"
+            app:layout_constraintEnd_toStartOf="@+id/iv_cast_image_4"
+            app:layout_constraintStart_toEndOf="@+id/iv_cast_image_2"
+            app:layout_constraintTop_toTopOf="@id/iv_cast_image_1" />
+
+        <ImageView
+            android:id="@+id/iv_cast_image_4"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_4"
+            app:layout_constraintEnd_toStartOf="@id/right_guidline"
+            app:layout_constraintStart_toEndOf="@+id/iv_cast_image_3"
+            app:layout_constraintTop_toTopOf="@id/iv_cast_image_1" />
+
+        <TextView
+            android:id="@+id/tv_cast_imagine_text_1"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="6dp"
+            android:text="@string/robert_downey_jr"
+            android:textColor="@color/white"
+            android:textSize="12sp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toStartOf="@+id/tv_cast_imagine_text_2"
+            app:layout_constraintStart_toStartOf="@id/iv_cast_image_1"
```

Comment:
relate left and right borders to iv_cast_image_1
and same for other actor's names: relate them to images above